centreon/www/front_src/src/Authentication/Authentication.cypress.spec.tsx

```diff
@@ -0,0 +1,117 @@
+
+import { authenticationProvidersEndpoint } from './api/endpoints';
+import { Provider } from './models';
+import { defaultPasswordSecurityPolicyAPI } from './Local/defaults';
+import { retrievedOpenidConfiguration } from './Openid/defaults';
+import { retrievedWebSSOConfiguration } from './WebSSO/defaults';
+import { retrievedSAMLConfiguration } from './SAML/defaults';
+import { labelOpenIDConnectConfiguration } from './Openid/translatedLabels';
+import { labelWebSSOConfiguration } from './WebSSO/translatedLabels';
+import { labelSAMLConfiguration } from './SAML/translatedLabels';
+import {
+  labelAuthenticationConditions,
+  labelAutoImportUsers,
+  labelGroupsMapping,
+  labelIdentityProvider
+} from './translatedLabels';
+import { labelRolesMapping } from './shared/translatedLabels';
+
+import Authentication from '.';
+
+dayjs.extend(duration);
+
+const initializeAndMountAuthenticationPage = (): void => {
+  cy.interceptAPIRequest({
+    alias: 'getLocalAuthenticationConfiguration',
+    method: Method.GET,
+    path: replace('./', '**', authenticationProvidersEndpoint(Provider.Local)),
+    response: defaultPasswordSecurityPolicyAPI
+  });
+  cy.interceptAPIRequest({
+    alias: 'getOpendidConnectConfiguration',
+    method: Method.GET,
+    path: replace('./', '**', authenticationProvidersEndpoint(Provider.Openid)),
+    response: retrievedOpenidConfiguration
+  });
+  cy.interceptAPIRequest({
+    alias: 'getWebSSOConfiguration',
+    method: Method.GET,
+    path: replace('./', '**', authenticationProvidersEndpoint(Provider.WebSSO)),
+    response: retrievedWebSSOConfiguration
+  });
+  cy.interceptAPIRequest({
+    alias: 'getSAMLConfiguration',
+    method: Method.GET,
+    path: replace('./', '**', authenticationProvidersEndpoint(Provider.SAML)),
+    response: retrievedSAMLConfiguration
+  });
+
+  cy.mount({
+    Component: (
+      <TestQueryProvider>
+        <Authentication />
+      </TestQueryProvider>
+    )
+  });
+
+  cy.viewport('macbook-13');
+};
+
+describe('Authentication configuration', () => {
+  beforeEach(() => {
+    initializeAndMountAuthenticationPage();
+  });
+
+  it('displays the local authentication tab by default', () => {
+    cy.waitForRequest('@getLocalAuthenticationConfiguration');
+
+    cy.matchImageSnapshot();
+  });
+
+  it('displays the Openid Connect configuration tab when the "Openid Connect configuration" tab is clicked', () => {
+    cy.waitForRequest('@getLocalAuthenticationConfiguration');
+
+    cy.findByText(labelOpenIDConnectConfiguration).click();
+
+    cy.waitForRequest('@getOpendidConnectConfiguration');
+
+    cy.findByText(labelGroupsMapping).click();
+    cy.findByText(labelRolesMapping).click();
+    cy.findByText(labelAutoImportUsers).click();
+    cy.findByText(labelAuthenticationConditions).click();
+    cy.findByText(labelIdentityProvider).click();
+
+    cy.findByText(labelOpenIDConnectConfiguration).scrollIntoView();
+
+    cy.matchImageSnapshot();

```

Comment:
I think as a general rule, it's nice to have tests that describes the case more in detail before snapshoting.
snapshotting make sure the design is ISO. 
test case make sure you have the right structure, and elements are correctly labeled and semantic etc...

For exemple, if the test opens a tab with some éléments inside :

```ts
cy.findByRole('region', { name: labelOpenIDConnectConfiguration })
  .as('getOpendidConnectConfiguration')
  .should('have.attr', 'aria-expanded', 'true')
  .get('form').within(() => {
      cy.findByLabelText(labelGroupsMapping).should('be.visible')
      cy.findByLabelText(labelAutoImportUsers).should('be.visible')
  ...etc
})


cy.get('@ getOpendidConnectConfiguration').scrollIntoView();
cy.matchImageSnapshot();
```

just as an exemple, I don't know your exact structure. 

here : 
`cy.findByRole('region', { name: labelOpenIDConnectConfiguration })` make sure the content of the tab is labelled by the button

`.should('have.attr', 'aria-expanded', 'true')` expect the screen readers to be able to detect if the region is visible or not

 `cy.findByLabelText(` makes sure elements are inputs and correctly labeled.


I don't get why you need to click on so many elements to fulfill the test case ('displays the Openid Connect configuration when the corresponding tab is clicked'). or maybe the case name lacks description